specs/simple-serialize.md

```diff
@@ -51,18 +57,22 @@ For convenience we alias:
 * `"byte"` to `"uint8"` (this is a basic type)
 * `"bytes"` to `["byte"]` (this is *not* a basic type)
 * `"bytesN"` to `["byte", N]` (this is *not* a basic type)
+* `"null"`: `{}`, i.e. the empty container
 
 ### Default values
```

Comment:
Default values for `union` types are not defined.

Reply:
Good point, I would suggest making it the the first type in the union, and requiring that if null is present, it should be the first type.